gammapy/data/tests/test_observations.py

```diff
@@ -628,3 +618,29 @@ def test_event_setter():
     events = EventList(Table())
     obs.events = events
     assert obs.events is events
+
+
+@requires_data()
+def test_observations_getitem(data_store):
+    """Test mask indexing on Observations"""
+    obs_1 = data_store.get_observations([20136, 20137, 20151])
+
+    assert isinstance(obs_1[0], Observation)
```

Comment:
This cover the case with numpy array right?

Reply:
Line 638-641 does it!